packages/vscode-extension/src/core/SuiteCloudRunner.ts

```diff
@@ -6,13 +6,14 @@
 import VSConsoleLogger from '../loggers/VSConsoleLogger';
 import { sdkPath } from './sdksetup/SdkProperties';
 
-import { CommandActionExecutor, CommandOptionsValidator, CLIConfigurationService } from '../util/ExtensionUtil';
+import { CommandActionExecutor, CommandOptionsValidator, CLIConfigurationService, ApplicationConstants } from '../util/ExtensionUtil';
 import CommandsMetadataSingleton from '../service/CommandsMetadataSingleton';
 
 export default class SuiteCloudRunner {
 	private commandActionExecutor: any;
 
 	constructor(executionPath?: string) {
+		process.argv.push(`${ApplicationConstants.PROJECT_FOLDER_ARG}=${executionPath}`);
```

Comment:
Why this line does make it work in Linux?

Reply:
Actually, RC checked and it doesn't work in Windows either, so it's not a Linux specific bug. Also, I broke it myself with the last refactor (shame on me!)